includes/class-wsuwp-graduate-degree-programs.php

```diff
@@ -354,8 +595,8 @@ public function display_factsheet_primary_meta_box( $post ) {
 				}
 
 				?>
-					<script type="text/template" id="factsheet-deadline-template">
-						<span class="factsheet-<?php echo esc_attr( $meta['type'] ); ?>-field">
+				<script type="text/template" id="factsheet-deadline-template">
+					<span class="factsheet-<?php echo esc_attr( $meta['type'] ); ?>-field">
 							<select name="<?php echo esc_attr( $key ); ?>[<%= form_count %>][semester]">
```

Comment:
Certainly not critical, but it looks this and the following few lines should be outdented by one tab, too.